package.json

```diff
@@ -20,6 +20,7 @@
     "react-bootstrap-table2-paginator": "^2.1.2",
     "react-calendar": "^3.2.1",
     "react-dom": "^16.13.1",
+    "react-responsive": "^9.0.0-beta.4",
```

Comment:
We already have a media-query react hook, so let's not add a new package for it. See https://github.com/codeforpdx/dwellingly-app/blob/37fdc7c0b5e65985e8e75be01e0f0faf32d24f51/src/views/managers/managers.jsx#L101 for an example